src/components/LunchboxWrapper/prepCanvas.ts

```diff
@@ -2,21 +2,37 @@ import { MiniDom } from '../../core'
 import { Ref } from 'vue'
 import { resizeCanvas } from './resizeCanvas'
 
+const getInnerDimensions = (node: Element) => {
+    const computedStyle = getComputedStyle(node)
+    const width = node.clientWidth - parseFloat(computedStyle.paddingLeft) - parseFloat(computedStyle.paddingRight)
+    const height = node.clientHeight - parseFloat(computedStyle.paddingTop) - parseFloat(computedStyle.paddingBottom)
+    return { width, height }
+}
+
 export const prepCanvas = (
     container: Ref<MiniDom.RendererDomNode | undefined>,
     canvasElement: HTMLCanvasElement,
     onBeforeUnmount: Function,
+    sizePolicy?: String,
```

Comment:
See note on `SizePolicy` type in `types.ts` - could you replace `String` with `SizePolicy` as well once it's added?